src/index.html

```diff
@@ -1,12 +1,29 @@
 <!doctype html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+  <meta charset="UTF-8" />
   <title>Systers Open Source</title>
   <base href="/">
-
-  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta charset="UTF-8" />
```

Comment:
this line appears to be duplicate of line #4.

Reply:
@bethanyr I will remove it.